File: Arreglos/TiposEliminacion/Indice0.js
//Programa para eliminar un elemento al principio del array
//Autor: Rivera Cota Jesús Crispín
//FACULTAD DE INGENIERÍA LOS MOCHIS
//Ing. de Software 201

const array = [11, 21, 31, 41, 51, 61]
//console.log("Antes de la eliminación, el array es: ", array)
//array.splice(0, 1) // Elimina el elemento en el índice 1
//console.log("Después de la eliminación, el array es: ", array)
console.log("Antes de la eliminación, el array es: ")
for (let i = 0; i < array.length; i++){
    console.log(array[i])
}
array = array.slice(0, 1); // Elimina el elemento en el índice 1
console.log("Después de la eliminación, el array es: ")
for (let i = 0; i < array.length; i++){
    console.log(array[i])
}

File: Arreglos/TiposEliminacion/Indice0.java
//Programa para eliminar un elemento al principio del array
//Autor: Rivera Cota Jesús Crispín
//FACULTAD DE INGENIERÍA LOS MOCHIS
//Ing. de Software 201

import java.util.Arrays;

public class TiposElim {
    public static void main(String[] args) {
        int[] array = {11, 21, 31, 41, 51, 61};
        System.out.println("El array antes de eliminar el elemento es: ");
        for (int i = 0; i < array.length; i++) {
            System.out.print(array[i] + " ");
        }
        array = Arrays.copyOfRange(array, 1, array.length);
        System.out.println("\nDespúes de la eliminación, el array es: ");
        for (int i = 0; i < array.length; i++) {
            System.out.print(array[i] + " ");
        }
    }
}

